include/slang/driver/Driver.h

```diff
@@ -331,6 +331,11 @@ class SLANG_EXPORT Driver {
     /// Prints all macros from all loaded buffers to stdout.
     void reportMacros();
 
+    std::vector<std::filesystem::path> getDepFiles(bool includesOnly = false) const;
+
+    std::string writeDepfiles(const std::vector<std::filesystem::path>& files,
```

Comment:
I don't love the name because it's not actually writing to anything anymore. Perhaps something like "serializeDepFile" instead? Not great either to be honest but maybe better. Also should be consistent that "DepFile" is two "words", like in getDepFiles above.